### ISSUE TITLE
C:\actions-runner\_work\pytorch\pytorch\builder\windows\pytorch\aten\src\ATen\native\cuda\TensorCompare.cu:110: block: [0,0,0], thread: [0,0,0] Assertion `input[0] != 0` failed.

### ISSUE THREAD

**Issue by jinnaluo0** [EvilBT/ComfyUI_SLK_joy_caption_two#92]
got prompt
D:\ComfyUI_windows_portable\ComfyUI\models\clip\siglip-so400m-patch14-384
Loading VLM's custom vision model
Prompt: Write a long descriptive caption for this image in a formal tone.
Requested to load SiglipVisionTransformer
Loading 1 new model
loaded completely 0.0 809.1729736328125 True
Requested to load ImageAdapter
Loading 1 new model
loaded completely 0.0 41.0390625 True
Loading tokenizer
Loading LLM: unsloth/Meta-Llama-3.1-8B-Instruct-bnb-4bit
D:\ComfyUI_windows_portable\ComfyUI\models\LLM\Meta-Llama-3.1-8B-Instruct-bnb-4bit
Successfully modified 'base_model_name_or_path' value in 'D:\ComfyUI_windows_portable\ComfyUI\models\Joy_caption_two\text_model\adapter_config.json'.
Unused kwargs: ['_load_in_4bit', '_load_in_8bit', 'quant_method']. These kwargs are not used in <class 'transformers.utils.quantization_config.BitsAndBytesConfig'>.
Starting from v4.46, the `logits` model output will have the same type as the model (except at train time, where it will always be FP32)
C:\actions-runner\_work\pytorch\pytorch\builder\windows\pytorch\aten\src\ATen\native\cuda\TensorCompare.cu:110: block: [0,0,0], thread: [0,0,0] Assertion `input[0] != 0` failed.
Fatal Python error: Aborted

Thread 0x00004d04 (most recent call first):
  File "concurrent\futures\thread.py", line 81 in _worker
  File "threading.py", line 982 in run
  File "threading.py", line 1045 in _bootstrap_inner
  File "threading.py", line 1002 in _bootstrap

Thread 0x00000674 (most recent call first):
  File "concurrent\futures\thread.py", line 81 in _worker
  File "threading.py", line 982 in run
  File "threading.py
"
, line 1045 in _bootstrap_inner
  File "threading.py", line 1002 in _bootstrap

Current thread 0x00004fd4 (most recent call first):
  File "D:\ComfyUI_windows_portable\python_embeded\Lib\site-packages\transformers\generation\utils.py"
, line 3044 in _sample
  File "D:\ComfyUI_windows_portable\python_embeded\Lib\site-packages\transformers\generation\utils.py", line 2048 in generate
  File "D:\ComfyUI_windows_portable\python_embeded\Lib\site-packages\torch\utils\_contextlib.py", line 116 in decorate_context
  File "D:\ComfyUI_windows_portable\ComfyUI\custom_nodes\ComfyUI_SLK_joy_caption_two\joy_caption_two_node.py", line 569 in generate
  File "D:\ComfyUI_windows_portable\ComfyUI\execution.py", line 158 in process_inputs
  File "D:\ComfyUI_windows_portable\ComfyUI\execution.py", line 169 in _map_node_over_list
  File "D:\ComfyUI_windows_portable\ComfyUI\execution.py", line 198 in get_output_data
  File "D:\ComfyUI_windows_portable\ComfyUI\execution.py", line 323 in execute
  File "D:\ComfyUI_windows_portable\ComfyUI\execution.py
", line 500
 in execute
  File "D:\ComfyUI_windows_portable\ComfyUI\main.py", line 126 in 
prompt_worker

  File "threading.py", line 982 in run
  File "threading.py", line 1045 in _bootstrap_inner
  File "threading.py", line 1002 in _bootstrap

Thread 0x00004118 (most recent call first):
  File "selectors.py", line 314 in _select
  File "selectors.py", line 323 in select
  File "asyncio\base_events.py", line 1884 in _run_once
  File "
asyncio\base_events.py
", line 607 in run_forever
  File "asyncio\base_events.py", line 640 in run_until_complete
  File "D:\ComfyUI_windows_portable\ComfyUI\main.py"
, line 
262 in <module>

Extension modules: yaml._yaml, numpy.core._multiarray_umath, numpy.core._multiarray_tests, numpy.linalg._umath_linalg, numpy.fft._pocketfft_internal, numpy.random._common
, numpy.random.bit_generator, 
numpy.random._bounded_integers
, numpy.random._mt19937, numpy.random.mtrand, numpy.random._philox, numpy.random._pcg64, numpy.random._sfc64, numpy.random._generator, torch._C, torch._C._dynamo.autograd_compiler, torch._C._dynamo.eval_frame, torch._C._dynamo.guards
, torch._C._dynamo.utils
, 
torch._C._fft
, torch._C._linalg, torch._C._nested, torch._C._nn, 
torch._C._sparse, torch._C._special
, PIL._imaging, psutil._psutil_windows, scipy._lib._ccallback_c, scipy.linalg._fblas, scipy.linalg._flapack, scipy.linalg.cython_lapack, scipy.linalg._cythonized_array_utils
, 
scipy.linalg._solve_toeplitz
, scipy.linalg._decomp_lu_cython, scipy.linalg._matfuncs_sqrtm_triu, scipy.linalg.cython_blas, scipy.linalg._matfuncs_expm
, scipy.linalg._decomp_update
, scipy.sparse._sparsetools, _csparsetools
, 
scipy.sparse._csparsetools, scipy.sparse.linalg._dsolve._superlu, scipy.sparse.linalg._eigen.arpack._arpack, scipy.sparse.linalg._propack._spropack
, 
scipy.sparse.linalg._propack._dpropack
, scipy.sparse.linalg._propack._cpropack, scipy.sparse.linalg._propack._zpropack, 
scipy.sparse.csgraph._tools, scipy.sparse.csgraph._shortest_path, scipy.sparse.csgraph._traversal, scipy.sparse.csgraph._min_spanning_tree, scipy.sparse.csgraph._flow, scipy.sparse.csgraph._matching, scipy.sparse.csgraph._reordering, scipy.optimize._minpack2, scipy.optimize._group_columns, scipy._lib.messagestream
, scipy.optimize._trlib._trlib, scipy.optimize._lbfgsb
, 
_moduleTNC
, scipy.optimize._moduleTNC, scipy.optimize._cobyla, 
scipy.optimize._slsqp, scipy.optimize._minpack
, scipy.optimize._lsq.givens_elimination
, scipy.optimize._zeros
, scipy.optimize._highs.cython.src._highs_wrapper, scipy.optimize._highs._highs_wrapper, scipy.optimize._highs.cython.src._highs_constants
, 
scipy.optimize._highs._highs_constants
, scipy.linalg._interpolative, scipy.optimize._bglu_dense
, scipy.optimize._lsap
, scipy.spatial._ckdtree, scipy.spatial._qhull, scipy.spatial._voronoi
, scipy.spatial._distance_wrap, scipy.spatial._hausdorff, scipy.special._ufuncs_cxx, scipy.special._cdflib, scipy.special._ufuncs, scipy.special._specfun, 
scipy.special._comb, scipy.special._ellip_harm_2
, scipy.spatial.transform._rotation, scipy.optimize._direct, _win32sysloader
, win32api
, PIL._imagingft, av._core
, av.logging
, av.bytesource, av.buffer, 
av.audio.format, av.enum, av.error, av.utils, av.option, av.descriptor, av.container.pyio, av.dictionary
, av.format
, av.stream, av.container.streams, av.sidedata.motionvectors, av.sidedata.sidedata, av.packet
, 
av.container.input, av.container.output, av.container.core, av.codec.context
, av.video.format
, av.video.reformatter, av.plane, av.video.plane
, av.video.frame
, av.video.stream, av.codec.codec, av.frame, av.audio.layout, av.audio.plane, av.audio.frame, av.audio.stream, av.audio.fifo, av.filter.pad, av.filter.link, av.filter.context, av.filter.graph
, av.filter.filter
, av.audio.resampler, charset_normalizer.md, markupsafe._speedups, sentencepiece._sentencepiece, regex._regex, scipy.integrate._odepack, scipy.integrate._quadpack, scipy.integrate._vode, scipy.integrate._dop, scipy.integrate._lsoda, scipy.ndimage._nd_image
, _ni_label, scipy.ndimage._ni_label, scipy.special.cython_special, 
scipy.stats._stats, scipy.stats.beta_ufunc
, scipy.stats._boost.beta_ufunc, scipy.stats.binom_ufunc, 
scipy.stats._boost.binom_ufunc, scipy.stats.nbinom_ufunc
, 
scipy.stats._boost.nbinom_ufunc, scipy.stats.hypergeom_ufunc, scipy.stats._boost.hypergeom_ufunc, scipy.stats.ncf_ufunc
, 
scipy.stats._boost.ncf_ufunc, scipy.stats.ncx2_ufunc
, scipy.stats._boost.ncx2_ufunc, scipy.stats.nct_ufunc, scipy.stats._boost.nct_ufunc, scipy.stats.skewnorm_ufunc
, scipy.stats._boost.skewnorm_ufunc, scipy.stats.invgauss_ufunc
, scipy.stats._boost.invgauss_ufunc, scipy.interpolate._fitpack, scipy.interpolate.dfitpack, scipy.interpolate._bspl
, scipy.interpolate._ppoly, scipy.interpolate.interpnd, scipy.interpolate._rbfinterp_pythran, scipy.interpolate._rgi_cython
, scipy.stats._biasedurn
, 
scipy.stats._levy_stable.levyst, scipy.stats._stats_pythran, scipy._lib._uarray._uarray, scipy.stats._ansari_swilk_statistics, scipy.stats._sobol
, 
scipy.stats._qmc_cy, scipy.stats._mvn, scipy.stats._rcont.rcont, scipy.stats._unuran.unuran_wrapper
, multidict._multidict, yarl._quoting_c
, aiohttp._helpers
, aiohttp._http_writer, aiohttp._http_parser, aiohttp._websocket
, frozenlist._frozenlist, _cffi_backend

--------

[程序异常退出，退出代码为 3 (0x00000003)]

**Comment by jinnaluo0** [comment]
** Platform: Windows
** Python version: 3.11.6 (tags/v3.11.6:8b6ee5b, Oct  2 2023, 14:57:12) [MSC v.1935 64 bit (AMD64)]

** Python executable: D:\ComfyUI_windows_portable\python_embeded\python.exe
** ComfyUI Path: D:\ComfyUI_windows_portable\ComfyUI

** Log path: D:\ComfyUI_windows_portable\ComfyUI\comfyui.log
[ComfyUI-Manager] Windows event loop policy mode enabled

Prestartup times for custom nodes:
   2.7 seconds: D:\ComfyUI_windows_portable\ComfyUI\custom_nodes\ComfyUI-Manager

Total VRAM 12287 MB, total RAM 32700 MB
pytorch version: 2.5.1+cu124
xformers version: 0.0.28.post3
Set vram state to: NORMAL_VRAM
Device: cuda:0 NVIDIA GeForce RTX 3060 : cudaMallocAsync
Using xformers cross attention

**Comment by jinnaluo0** [comment]
执行python -m pip install transformers==4.44.0后：

 Attempting uninstall: transformers
    Found existing installation: transformers 4.45.1
    Uninstalling transformers-4.45.1:
      Successfully uninstalled transformers-4.45.1
  WARNING: The script transformers-cli.exe is installed in 'D:\ComfyUI_windows_portable\python_embeded\Scripts' which is
 not on PATH.
  Consider adding this directory to PATH or, if you prefer to suppress this warning, use --no-warn-script-location.
ERROR: pip's dependency resolver does not currently take into account all the packages that are installed. This behaviou
r is the source of the following dependency conflicts.
image-reward 1.5 requires fairscale==0.4.13, but you have fairscale 0.4.0 which is incompatible.
Successfully installed tokenizers-0.19.1 transformers-4.44.0

有了新的错误：

!!! Exception during processing !!! `.to` is not supported for `4-bit` or `8-bit` bitsandbytes models. Please use the model as it is, since the model has already been set to the correct devices and casted to the correct `dtype`.

**Comment by qwaaws** [comment]
可以使用插件的低现存模式看看

**Comment by zl-comment** [comment]
我也是这个错误
